From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 4 May 2025 13:18:28 +0100
Subject: rm setchapternewpage

The two @setchapternewpage commands yield a warning.

 sml-mode.texi:66: warning: multiple @setchapternewpage

The documentation (texinfo setsetchapternewpage) says

 We recommend not including any @setchapternewpage command in your document source at all

so that's what this patch is doing.
---
 sml-mode.texi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sml-mode.texi b/sml-mode.texi
index e3df1fb..408f960 100644
--- a/sml-mode.texi
+++ b/sml-mode.texi
@@ -27,7 +27,6 @@
 @direntry
 * SML mode: (sml-mode).	        Emacs mode for editing SML
 @end direntry
-@setchapternewpage on
 
 @titlepage
 @sp 5
@@ -63,7 +62,6 @@ Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
 
 @end titlepage
 
-@setchapternewpage off
 @headings double
 
 @c ============================================================ TOP NODE
